### DISCUSSION TITLE
Collections + icon

### DISCUSSION THREAD

**Discussion by gitxvignesh** [Icon-Shelf/icon-shelf#148] [General]
Hey dude,
                  Icon Shelf is great. But for the collection icon and for All Icons, there is a lot of space. Add the + to the collection row So it would be look nice dude

**Comment by MrRobz** [comment]
These are great suggestions. Thanks for the feedback.
If you have time, feel free to give a Pull request for this change.

**Comment by MrRobz** [comment]
Closed by https://github.com/Icon-Shelf/icon-shelf/pull/149
